components/camel-platform-http-vertx/pom.xml

```diff
@@ -84,6 +84,11 @@
             <artifactId>camel-log</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-vertx-http</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
I'm a bit concerned that this may cause some weird cyclic dependency problems (or at least make the whole dependency chain more complex).

Reply:
Yeah potentially - though it may be that camel-platform-http-vertx is built before all the vertx components where this one is from. But agree we had been bitten in the past with build cycles going full circle and breaking the build

Reply:
Updated the test to use `camel-http` instead.

Reply:
Thanks, but that still may create a very complex dependency chain as you have components within `components` with dependencies among themselves. 

The best way, I think, would be to create a `test-infra` module that is completely independent.

Reply:
Thanks for that, let me look into it.

Merry Christmas!

Reply:
Hey @orpiske, by using `camel-http` instead of `camel-vertx-http` it now means that no new deps have been introduced by this change. Would it be OK it we tackle the test dependency management improvements that you've proposed in a separate ticket/PR?

Reply:
Yes, that's OK to me. Thanks!